include/logger/loggerConfig.h

```diff
@@ -260,9 +260,9 @@ typedef struct _PWMConfig{
 } PWMConfig;
 
 /// PWM frequency in Hz.
-#define MAX_PWM_CLOCK_FREQUENCY             200000
-#define MIN_PWM_CLOCK_FREQUENCY	 	10
-#define DEFAULT_PWM_CLOCK_FREQUENCY			10000
+#define MAX_PWM_CLOCK_FREQUENCY             65000
```

Comment:
https://www.kernel.org/doc/Documentation/CodingStyle : Chapter 12

Please make it one space between the end of the definition and the value

```#define MAX_PWM_CLOCK_FREQUENCY 65000```